### PR TITLE
samples/order: fix on windows

### PR DESCRIPTION
**Pull request by jba**
WIP do not review

### CODE REVIEW

**Comment by jba** [comment]
This is ready for review.

File: samples/gocdk-runtimevar/runtimevar.ct [hunk 1]
```diff
@@ -22,12 +22,15 @@ $ echof fvar contents
 $ gocdk-runtimevar cat file:///${ROOTDIR}/fvar?decoder=string
 (string) contents
 
+$ echo ROOTDIR is '${ROOTDIR}'
```
Comment (vangent) [inline samples/gocdk-runtimevar/runtimevar.ct]: This can be removed?

File: samples/gocdk-blob/blob.ct [hunk 2]
```diff
@@ -18,19 +18,19 @@
 $ mkdir bucket
 
 # List an empty bucket: valid, but no output.
-$ gocdk-blob ls file://${ROOTDIR}/bucket
+$ gocdk-blob ls file://${SLASHDIR}/bucket
```
Comment (vangent) [inline samples/gocdk-blob/blob.ct]: This took me a bit to follow. Might it be more clear to use `ROOTDIR_URL` as the environment variable that is added in `Setup`, where the value is the full URL? Then this line, for example, would be:

```
$ gocdk-blob ls ${ROOTDIR_URL}/bucket
```

Up to you.

Comment (jba) [reply samples/gocdk-blob/blob.ct]: done

File: samples/gocdk-runtimevar/runtimevar.ct (hunk 1 above)
Comment (jba) [reply samples/gocdk-runtimevar/runtimevar.ct]: done

**Pull request merged**